### ISSUE TITLE
Corrupted workbook when using template with Data Bars and / or Icon Sets

### ISSUE THREAD

**Issue by Darxor** [ycphs/openxlsx#35]
If we load a workbook with data bars (or icon sets) set up and immediately save it, workbook will be corrupted. We do not really need to input any data whatsoever.

Upon opening freshly saved workbook and attempting to recover it, there will be an error "HRESULT 0x8000ffff". Workbook will be recovered, but many aspects of formatting will be stripped.

I am able to reproduce it in current CRAN version 4.1.4 and development version from this repo. However, it works fine with an older 4.1.3 version from CRAN.

This is not the same as #18 (but may be related), since this bug is introduced with latest version and sparklines are failing in 4.1.3 and with different error message upon recovering.

```
library(openxlsx)

wb_test <- loadWorkbook("test_wb.xlsx")

saveWorkbook(wb_test, "test_wb_openxlsx.xlsx", overwrite = TRUE)
openXL("test_wb_openxlsx.xlsx")
```

The template and resulting file:
[test_wb.xlsx](https://github.com/ycphs/openxlsx/files/4050580/test_wb.xlsx)
[test_wb_openxlsx.xlsx](https://github.com/ycphs/openxlsx/files/4050579/test_wb_openxlsx.xlsx)

<details><summary>Session info</summary>
<pre><code>
R version 3.6.2 (2019-12-12)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 18363)

Matrix products: default

locale:
[1] LC_COLLATE=Russian_Russia.1251  LC_CTYPE=Russian_Russia.1251    LC_MONETARY=Russian_Russia.1251 LC_NUMERIC=C                    LC_TIME=Russian_Russia.1251    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] openxlsx_4.1.3

loaded via a namespace (and not attached):
 [1] Rcpp_1.0.3             rstudioapi_0.10.0-9002 magrittr_1.5           usethis_1.5.1          devtools_2.2.1         pkgload_1.0.2          R6_2.4.1               rlang_0.4.2           
 [9] fansi_0.4.1            tools_3.6.2            pkgbuild_1.0.6         sessioninfo_1.1.1      cli_2.0.1              withr_2.1.2            ellipsis_0.3.0         remotes_2.1.0         
[17] assertthat_0.2.1       digest_0.6.23          rprojroot_1.3-2        crayon_1.3.4           zip_2.0.4              processx_3.4.1         callr_3.4.0            fs_1.3.1              
[25] ps_1.3.0               testthat_2.3.1         memoise_1.1.0          glue_1.3.1             compiler_3.6.2         desc_1.2.0             backports_1.1.5        prettyunits_1.1.0 
</code></pre>
</details>

I am also using Russian version of Excel 365 if that matters.

**Comment by ycphs** [comment]
#18 

Please check the linked issue.

**Comment by Darxor** [comment]
> #18
> 
> Please check the linked issue.

Sorry, but I did check it, and as mentioned above, they are not the same, albeit may be related. They appear in different versions of openxlsx and provoke different errors upon recovery in excel.

**Comment by ycphs** [comment]
Maybe this would be a more suitable solution.

````
file.rename("test_wb.xlsx","test_wb_openxlsx.xlsx")
# or 
file.copy("test_wb.xlsx","test_wb_openxlsx.xlsx")
````
 

> We do not really need to input any data whatsoever.


I see that it should be fixed, as it seems related to #18 we should work on a solution for both.

**Comment by Darxor** [comment]
> Maybe this would be a more suitable solution.

For this dummy case - yes it will work for sure, I just tried to simplify the example as much as I can.

So far I managed to track down commit that introduced this regression: https://github.com/ycphs/openxlsx/commit/40af512d090aff7878bb462137e8eb9ff63e5e10, but I honestly undestand too little about whats going on there.

**Comment by Darxor** [comment]
I think I found what was causing this issue. See PR #37 
It does not unfortunately fix #18 but its something
